hack/util.sh

```diff
@@ -65,6 +65,17 @@ function util::cmd_must_exist_cfssl {
     fi
 }
 
+# util::install_kubectl will install the given version kubectl
+function util::install_kubectl {
+    KUBECTL_VERSION=${1}
```

Comment:
Maybe it's better to declare local variables here likes `local KUBECTL_VERSION=${1}`  rather than global ones because the name 'KUBECTL_VERSION' and 'ARCH' are so common to redefine

Reply:
ok